AUTHORS

```diff
@@ -30,4 +30,4 @@ Nathan Schroer <schroer.14@wright.edu>
 Hayden Richter <richter.20@wright.edu>
 Jacob Smith <smith.2198@wright.edu>
 Sara Walsh <uglypurses@yahoo.com>
-
+Ian Smith <smith.1903@wright.edu>
```

Comment:
Please read the comment at the top of AUTHORS. You did not add your name in the correct place. Please respect the ordering.